### ISSUE TITLE
key with fn instead of keyword does not work

### ISSUE THREAD

**Issue by SohoPak** [clj-commons/pretty#125]
This code prints alright:
```(require '[clj-commons.format.table :refer [print-table]])
(print-table [{:key :foo
               :title "foo"}]
             [{:foo 42}
              {:foo 43}])

```

This one does not:
```
(require '[clj-commons.format.table :refer [print-table]])

(print-table [{:key (fn [x] (:foo x))
               :title "foo"}]
             [{:foo 42}
              {:foo 43}])
```

**Comment by hlship** [comment]
```
(print-table [{:key (fn [x] (:foo x))
                 :title "foo"}]
               [{:foo 42}
                {:foo 43}])
┏━━━━━┓
┃ foo ┃
┣━━━━━┫
┃     ┃
┃     ┃
┗━━━━━┛
```

**Comment by hlship** [comment]
Fixed by 301140258bf2aabf7d0e0dbaacd9a4cbf4fc8149
